manifests/params.pp

```diff
@@ -9,6 +9,10 @@
   $environment = undef
   $puppetmaster = "puppet.${::domain}"
 
+  $autosign = '/etc/puppetlabs/puppet/autosign.conf'
+  $autosign_runnable = false
+  $autosign_list     = []
+  $autosign_script   = ''
```

Comment:
This is causing the tests to fail.  Puppet 4 has an actual undef so using that value is the way to go.